controllers/users.js

```diff
@@ -1,7 +1,16 @@
+const User = require("../models/user");
+
 const UsersController = {
   New: (req, res) => {
     console.log("############## Now in UserController New ##############")
     res.render("users/new", {});
+  },
+
+  Create: (req, res) => {
+    console.log("############## Now in UserController Create ##############")
+    const user = new User(req.body);
+    user.save();
```

Comment:
`.save()` is async so I think we can make it wait until that is completed until redirecting

`
... .save().then(() => {
  res.status(201).redirect("/");
})
`